driver/csiplugin/connectors/rest_v2.go

```diff
@@ -549,23 +611,24 @@ func (s *SpectrumRestV2) UpdateFileset(ctx context.Context, filesystemName strin
 	return nil
 }
 
-func (s *SpectrumRestV2) CheckIfGatewayNodePresent(ctx context.Context) (bool, error) {
+func (s *SpectrumRestV2) CheckGatewayNodeDetails(ctx context.Context) (bool, string, error) {
```

Comment:
We can skip returning bool here, and can return gateway name and err. If the caller gets non-nil error and non-empty gateway name, that means gateway is present in the cluster. And the function can be renamed to `GetGatewayNode`

Reply:
done